cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java

```diff
@@ -1391,4 +1402,24 @@ private void refreshCapacity() {
       _clusterCapacity[r.id()] = capacity;
     }
   }
+
+  private boolean hasSameReplicasFor(Partition partition, PartitionInfo partitionInfo) {
```

Comment:
Nit: After the latest commit (i.e. now we are passing the `partition` as a parameter), we can make this function `static` and move it to the utility class of this package (see `ModelUtils`). So that it can be reused by other relevant classes that may want to make a similar comparison.